kolibri/plugins/management/assets/src/views/manage-content-page/wizard-export.vue

```diff
@@ -10,6 +10,10 @@
     <div class="main">
       <template v-if="!drivesLoading">
         <div class="modal-message">
+          <p>
+            {{ $tr('exportPromptPrefix', { numChannels: allChannels.length }) }}
+            ({{ exportContentSize }})
+          </p>
```

Comment:
Unfortunately, might need to combine `exportPromptPrefix` and `exportContentSize` so that the whole thing will be properly translated (e.g. in RTL languages).

Alternatively, might be worth discussing adding a 'subtext' prop to the `k-radio-button` component and incorporating it into the core style guide. I see that being used in other designs:

![image](https://user-images.githubusercontent.com/2367265/29148161-999f045a-7d20-11e7-94d6-e5784fd4f4ef.png)

cc @jtamiace @khangmach @christianmemije